lib/renderer/react.js

```diff
@@ -13,6 +13,7 @@ const reactExport = module.exports;
  * @param {string} templateLocation located for thetemplate
  * @param {string} templateContentDir where the template content are located
  * @param {string} transpiledTemplateLocation folder for the transpiled code
+ * @param {Boolean} compile Whether to compile the template files or used the cached transpiled version provided by the template in the '__transpiled' folder
```

Comment:
did you do some changes in below code? cause now this jsdoc is not reflecting reality 🤔